src/interface.jl

```diff
@@ -97,12 +97,32 @@ isrevertible(transform::Transform) =
   isrevertible(typeof(transform))
 isrevertible(::Type{<:Transform}) = false
 
+# revert does not need to be defined for non-revertible transforms
+function revert(transform::Transform, newobject, cache)
+  if !isrevertible(transform)
+    throw(ArgumentError("Can't revert the non-revertible transform $transform"))
+  end
+  return revert(transform, newobject, cache)
+end
```

Comment:
This should cause an infinite loop, right?

Reply:
Apparently, yes 😂 😂. My brain lagged there; let me fix that. If you like, I will replace that with another error saying that the transform is revertible but revert has not yet been implemented.

Same applies to inverse below.